INSTALL ERROR: Svelte 4 not a peer dependency?

Got this error when installing supakit:

```
% npm install supakit
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: lstv@0.0.1
npm ERR! Found: svelte@4.2.0
npm ERR! node_modules/svelte
npm ERR!   dev svelte@"^4.1.2" from the root project
npm ERR!   peer svelte@"^3.54.0 || ^4.0.0-next.0" from @sveltejs/kit@1.22.5
npm ERR!   node_modules/@sveltejs/kit
npm ERR!     dev @sveltejs/kit@"^1.22.4" from the root project
npm ERR!     peer @sveltejs/kit@"^1.20.4" from supakit@1.1.0
npm ERR!     node_modules/supakit
npm ERR!       supakit@"*" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer svelte@"^3.59.1" from supakit@1.1.0
npm ERR! node_modules/supakit
npm ERR!   supakit@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /Users/rchrdnsh/.npm/_logs/2023-08-11T18_35_26_993Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/rchrdnsh/.npm/_logs/2023-08-11T18_35_26_993Z-debug-0.log
```

...pleez halp?

--- a/package.json
+++ b/package.json
@@ -27,7 +27,7 @@
 	"peerDependencies": {
 		"@supabase/supabase-js": "^2.26.0",
 		"@sveltejs/kit": "^1.20.4",
-		"svelte": "^3.59.1"
+		"svelte": ">=3.59.1"
 	},
 	"type": "module",
 	"exports": {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "supakit",
-	"version": "1.1.0",
+	"version": "1.2.0",
 	"scripts": {
 		"dev": "vite dev",
 		"build": "svelte-kit sync && svelte-package",

